hms-api/src/rest/ideaComment/IdeaCommentEditRequest.ts

```diff
@@ -0,0 +1,18 @@
+import Uuid from '../../util/Uuid';
+
+class IdeaCommentEditRequest {
+  id: Uuid;
```

Comment:
I don't think you actually use this anywhere since we just get it from the path.  The other editXXXRequest objects don't have them either.